### DISCUSSION TITLE
Cant install the package

### DISCUSSION THREAD

**Discussion by Lolmc0587** [CoolCat467/fix-lwjgl#29] [Q&A] [answered]
Defaulting to user installation because normal site-packages is not writeable
Collecting git+https://github.com/CoolCat467/fix-lwjgl.git
  Cloning https://github.com/CoolCat467/fix-lwjgl.git to /tmp/pip-req-build-0dp_elgm
  Running command git clone --filter=blob:none --quiet https://github.com/CoolCat467/fix-lwjgl.git /tmp/pip-req-build-0dp_elgm
  Resolved https://github.com/CoolCat467/fix-lwjgl.git to commit 062266190fd3f0b63587178755759ab9d5861055
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: UNKNOWN
  Building wheel for UNKNOWN (pyproject.toml) ... done
  Created wheel for UNKNOWN: filename=UNKNOWN-0.0.0-py3-none-any.whl size=1793 sha256=11c1f2b1422a2a00762f74a53bc0d3c7170f897bf8cec28d634a8a0b9dd07f04
  Stored in directory: /tmp/pip-ephem-wheel-cache-16g6_nzy/wheels/4f/4a/20/40e3de3311fe0daf2616bfa03058b9a4be607c10eaa19b3370
Successfully built UNKNOWN
Installing collected packages: UNKNOWN
Successfully installed UNKNOWN-0.0.0

**Comment by CoolCat467** [comment, marked as answer]
@Lolmc0587 Could you re-create this as a new issue? [Create bug report link](https://github.com/CoolCat467/fix-lwjgl/issues/new?template=bug_report.md)

**Reply by CoolCat467** [reply]
More likely than not, your pip installation is out of date, but I would need more information to be more sure. You can attempt to fix this by running
```console
pip install --upgrade pip
```
and then re-running the installation script.
If this doesn't work, again, please create a bug report and I can help you further from there.

**Reply by Lolmc0587** [reply]
i have found problem because i use pip 22.0.2 to install
now i upgrade to pip 24.0 and it work now
but now there are new error:
[log_launcher.txt](https://github.com/CoolCat467/fix-lwjgl/files/14626256/log_launcher.txt)
my setting:
![Screenshot from 2024-03-17 12-03-52](https://github.com/CoolCat467/fix-lwjgl/assets/59377673/042d5168-b50b-4260-b3f5-9a5b1a0f9cba)

**Reply by CoolCat467** [reply]
Moved to https://github.com/CoolCat467/fix-lwjgl/issues/30

**Reply by CoolCat467** [reply]
Reply in moved: https://github.com/CoolCat467/fix-lwjgl/issues/30#issuecomment-2002312644
